Makefile

```diff
@@ -1,9 +1,10 @@
 PKG = github.com/airbnb/rudolph
 VERSION := $(shell git describe --tags --always)
 DOCS_DIR ?= ./docs
-DEPLOYMENT_ZIP_PATH = $(PWD)/build/package/deployment.zip
+RUDOLPH_API_DEPLOYMENT_ZIP_PATH = $(PWD)/build/package/api_deployment.zip
```

Comment:
since each zip now needs to contain a singular binary called `bootstrap` this is required to split each `handler` into its own zip...